fmskill/metrics.py

```diff
@@ -410,6 +410,26 @@ def si(obs: np.ndarray, model: np.ndarray) -> float:
 def scatter_index(obs: np.ndarray, model: np.ndarray) -> float:
     """Scatter index (SI)
 
+    Which is the same as the unbiased-RMSE normalized by the mean of the observations. 
+
+    .. math::
+        \\sqrt {\frac{1}{n} \\sum_{i=1}^n \\left( (model_i - \\overline {model}) - (obs_i - \\overline {obs}) \\right)^2
+        {\frac{1}{n} \\sum_{i=1}^n obs_i^2}}
+
+    Range: [0, 100]; Best: 0
```

Comment:
I guess that the value is between 0 and 1, but the best value should be 0 (if BIAS=0 and model=measurement, which would a Perfect Model , then the Numerator is 0)

Reply:
I agree that the best value is 0, but the range is $(-\infty, \infty)$.

Reply:
I see, the denominator should be the `mean of the abs (x)` and then the range could not be negative.

Reply:
Better now? 

![image](https://user-images.githubusercontent.com/34088801/195292356-cc316c56-484f-49cb-8e65-944a68cbdaf6.png)

Reply:
sure, the tests are still failing though!

Reply:
That was why I asked for your input to test the implementation because you could compare it with your Matlab implementation... could you please do that?

Reply:
now I get it :)
for sure

Reply:
alright done, I checked against the other implementation and it gives the same result. 
I updated the test as well. For some reason though I pass all my tests with pytest, but it is failing on the  auto-tests

Reply:
> alright done, I checked against the other implementation and it gives the same result. I updated the test as well ( I had no idea tests could fail on the docstring as well! ) In this branch the new SI implementation becomes the default, overriding the old one

Take a look at [Doctest](https://docs.python.org/3/library/doctest.html) 🤯

Reply:
@daniel-caichac-DHI I think you are the only using this metric. (it doesn't make sense to anyone else😉)

📢 If anyone else is using this metric and wants to keep the other formulation, now is the time to speak up, otherwise it will be gone!

Reply:
_everyone_ I know is using this metric!
I will remove the draft then and move it into the soon to be merged pipeline